### PR TITLE
modified app service URL and added an additional field "workspace_name" to instance details

### PR DESCRIPTION
**Pull request by muralikarthikk**

### CODE REVIEW

File: appstore/api/v1/serializers.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ class InstanceSerializer(serializers.Serializer):
     aid = serializers.CharField(allow_null=True)
     sid = serializers.CharField()
     fqsid = serializers.CharField()
+    workspace_name = serializers.CharField()
```
Comment (n0mn0m) [inline appstore/api/v1/serializers.py]: Does this need to support `empty` ?

Comment (muralikarthikk) [reply appstore/api/v1/serializers.py]: Yes, I think it should support empty string. I will go ahead and make that change.

File: appstore/api/v1/serializers.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class InstanceSerializer(serializers.Serializer):
     aid = serializers.CharField(allow_null=True)
     sid = serializers.CharField()
     fqsid = serializers.CharField()
-    workspace_name = serializers.CharField()
+    workspace_name = serializers.CharField(allow_blank=True)
```
Comment (muralikarthikk) [inline appstore/api/v1/serializers.py:26]: @n0mn0m Greatly appreciate your input. Made this change, please let me know if this would suffice.

**Review by ismagilani — APPROVED** [review]

File: appstore/api/v1/serializers.py (hunk 2 above)
Comment (n0mn0m) [reply appstore/api/v1/serializers.py:26]: Looks good to me.

**Comment by n0mn0m** [comment]
@muralikarthikk I think the tycho PR needs to merge first, pulled this down and get test failures:

```
======================================================================
ERROR: test_logged_in_can_get_instance_list (api.v1.tests.TestInstanceView)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/alexanderh/projects/appstore/appstore/api/v1/tests.py", line 77, in test_logged_in_can_get_instance_list
    response = list_view(api_request)
  File "/Users/alexanderh/.virtualenvs/appstore/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/alexanderh/.virtualenvs/appstore/lib/python3.9/site-packages/rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/alexanderh/.virtualenvs/appstore/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/alexanderh/.virtualenvs/appstore/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/alexanderh/.virtualenvs/appstore/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/alexanderh/.virtualenvs/appstore/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/alexanderh/projects/appstore/appstore/api/v1/views.py", line 292, in list
    instance.workspace_name,
AttributeError: 'TychoService' object has no attribute 'workspace_name'

----------------------------------------------------------------------
Ran 24 tests in 7.027s

FAILED (errors=1)
```

**Comment by n0mn0m** [comment]
@muralikarthikk test are green with the version bump. I think you probably want to rebase off `develop` and this should be good to go.

**Pull request merged**